core/src/main/java/org/bitcoinj/core/FullPrunedBlockChain.java

```diff
@@ -164,18 +176,20 @@ public VerificationException call() throws Exception {
         }
     }
 
-    /** Get the {@link Script} from the script bytes or null if it doesn't parse. */
-    @Nullable
+    /**
+     * Get the {@link Script} from the script bytes or null if it doesn't parse.
```

Comment:
When I made my changes and ran the tests without changing the method I had gotten NullPointerException in H2FullPrunedBlockChainTest.AbstractFullPrunedBlockChainTest: 
org.bitcoinj.store.DatabaseFullPrunedBlockStore.addUnspentTransactionOutput(DatabaseFullPrunedBlockStore.java:965)

The error occurs because I started to use 
s.setBytes(5, out.getScript().getProgram());
instead of 
s.setBytes(5, out.getScriptBytes());

There are two methods in this class: getScript() and getScriptAddress() starts from 182 line. The last one is returning blank string if the script is null or script can't be parsed to address. If you analyze both methods using , you will find  they only hides an error and swallows the exceptions. I'd like to refactor it to avoid using the methods. I don't know how to deal with NullPointerException mentioned above. I assume there is something with test data. But the test is to complicated for me to make changes. 

So I made changes to getScript() to return not null. At least now method behaves exactly as getScriptAddress(). 

It's not only place where exceptions are swallowed. Check  AbstractFullPrunedBlockChainTest, line 116 

JavaDoc fixed.